README.md

```diff
@@ -12,8 +12,16 @@ The below usage instructions are for the combined package.
 
 ### Install it via NPM:
 
+#### Vue@^3
+
 ```shell
-npm install @dialpad/dialtone@latest
+npm install @dialpad/dialtone
+```
+
+#### Vue@2.6
+
+```shell
+npm install @dialpad/dialtone @dialpad/i18n-vue2
```

Comment:
we include this in dependencies but we also need to install it manually? Isn't this the same issue we had with tiptap before? Can we resolve this in the same way?

Reply:
This is included on `@dialpad/dialtone-vue` dependencies, it is required to install it independently while using `@dialpad/dialtone` package, I cannot add it to the package peer dependencies as it would cause issues with Vue@^3.

Also it can't be bundled within the dialtone-vue package as it uses vue-demi that checks the Vue version while bundling and we're using Vue@^2.7 on our mono repo which is not compatible with products Vue 2.6

This was the issue in this PR: https://github.com/dialpad/dialtone/pull/753/files